src/App.js

```diff
@@ -46,590 +53,453 @@ const languageProps = {
-          economicSchemas: economicSchemas,
-          hubSchemas: hubsSchemas,
-        };
-        this.setState({
-          sanitySchemas: sanitySchemas,
-        });
-        localStorage.setItem("sanity", JSON.stringify(sanitySchemas));
-      } else {
-        this.setState({ sanitySchemas: JSON.parse(existingSanityData) });
+    async function fetchSanitySchemas() {
+      try {
+        let existingSanityData = localStorage.getItem("sanity");
+        if (existingSanityData === null) {
+          const query = `*[_type == 'project']`;
+          const query1 = `*[_type == 'economic']`;
+          const query2 = `*[_type == 'hubs' && !(_id in path("drafts.**"))]`;
+          const [projectSchemas, economicSchemas, hubsSchemas] =
+            await Promise.all([
+              sanity.fetch(query),
+              sanity.fetch(query1),
+              sanity.fetch(query2),
+            ]);
+
+          let sanitySchemas = {
+            technicalSchemas: projectSchemas,
+            economicSchemas: economicSchemas,
+            hubSchemas: hubsSchemas,
+          };
+          setState({
+            ...state,
+            sanitySchemas: sanitySchemas,
+          });
+          localStorage.setItem("sanity", JSON.stringify(sanitySchemas));
+        } else {
+          setState({ ...state, sanitySchemas: JSON.parse(existingSanityData) });
+        }
+      } catch (e) {
+        errorToast(e);
       }
-    } catch (e) {
-      errorToast(e);
     }
-  };
+  }, []);
 
-  fetchMenteeSprints = async (userId) => {
+  const { putUpdatedSprintData } = props;
+
+  const fetchMenteeSprints = async (userId) => {
     try {
       let menteeSprints = await API.get("pareto", `/sprints/mentee/${userId}`);
-      this.setState({ sprints: menteeSprints });
+      setState({ ...state, sprints: menteeSprints });
     } catch (e) {
       errorToast(e);
     }
   };
 
-  fetchCoachingRoster = async (id) => {
-    try {
-      let athletes = await API.get("pareto", `/relationship/mentor/${id}`);
-      this.setState({ athletes: athletes });
-    } catch (e) {
-      console.log("Error fetching athletes");
-    }
+  const userHasAuthenticated = (authenticated) => {
+    setIsAuthenticated(authenticated);
   };
 
-  fetchCoaches = async (id) => {
-    try {
-      let existingCoaches = localStorage.getItem("coaches");
-      if (existingCoaches === null) {
-        let coaches = await API.get("pareto", `/relationship/mentee/${id}`);
-        this.setState({ coaches: coaches });
-        localStorage.setItem("coaches", JSON.stringify(coaches));
-      } else {
-        // check for empty arrays
-        this.setState({ coaches: JSON.parse(existingCoaches) });
-      }
-    } catch (e) {
-      console.log("Error fetching athletes");
-    }
-  };
-
-  userHasAuthenticated = (authenticated) => {
-    this.setState({ isAuthenticated: authenticated });
-  };
-
-  refreshExperience = (type, updatedObject) => {
+  const refreshExperience = (type, updatedObject) => {
```

Comment:
Everything from here on is very straightforward. Basically just adding const to the old class methods and changing this.state.xyzProperty to state.xyzProperty